CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## 1.0.0
```

Comment:
You can make these headers easily linkable by putting a section like this at the bottom of the document:

```md
[Unreleased]: /../../compare/v1.0.0...develop
[1.0.0]: /../../tree/v1.0.0
```

Reply:
That's a great idea! 👍

Reply:
It's based on the `markdown` example on https://keepachangelog.com/en/1.0.0/ by the way, but there they use the absolute URLs, where you can use relative URLs using the `/../../` trick.